content/03-reference/01-tools-and-interfaces/07-preview-features/001-mssql/020-mssql-local.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'MSSQL on Windows (local)'
+metaTitle: 'Microsoft SQL Connector (Reference)'
+metaDescription: 'A list of Prisma features currently in preview.'
+
+---
+
+<TopBlock>
+
+To run a MSSQL Server locally on a Windows machine:
+
+1. If you do not have access to an instance of MSSQL Server, download and set up [SQL Server 2019 Developer](https://www.microsoft.com/en-us/sql-server/sql-server-downloads).
+
+1. Download and install [SQL Server Management Studio](https://docs.microsoft.com/en-us/sql/ssms/download-sql-server-management-studio-ssms?view=sql-server-ver15).
+
+1. Use Windows Authentication to log in to MSSQL Server Management Studio (expand the **Server Name** dropdown and click **&lt;Browse for more...&gt;** to find your database engine):
+
+
+  ![The New Query button in MSSQL Management Studio](../../../../doc-images/connect-sql-server.png)
+
+</TopBlock>
+
+## Enable TCP/IP is enabled
+
+Prisma Client requires TCP/IP to be enabled. To enable TCP/IP:
+
+1. Open SQL Server Configuration Manager.
+
+1. In the left-hand panel, click **SQL Server Network Configuration** > **Protocols for MSSQLServer**
```

Comment:
Protocols for SQL Server

Reply:
Leaving this one as it's a straight copy from the UI